backoffice/templates/backoffice/home.html

```diff
@@ -3,6 +3,24 @@
 {% block heading %}Dashboard{% endblock %}
 
 {% block main %}
+  {% if drivers_pending > 0 or accounts_pending > 0 %}
+    <a href="/backoffice/approvals/" class="bg-yellow-300 flex-row hover:bg-yellow-100 m-2 p-2">
```

Comment:
Links should use the URL lookup template tag rather than being hard coded. I think the line below is correct for that.

```suggestion
    <a href="{% url backoffice_approvals %}" class="bg-yellow-300 flex-row hover:bg-yellow-100 m-2 p-2">
```

Reply:
You were close, it requires single quotes around the url name.